ui/src/views/Tasks/TaskLog/index.jsx

```diff
@@ -75,6 +77,7 @@ export default class TaskLog extends Component {
             </Link>
           }
         />
+        <TaskButtons data={this.props.data} url={url} rawButton />
```

Comment:
All the props that are defined here should be defined in the component itself in `propTypes` and perhaps `defaultProps` if any of those are not always required.